uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1021.php

```diff
@@ -41,4 +42,12 @@ private function up1021_setDefaultGenderOptions() {
     }
   }
 
+  private function up1021_disableDefaultGenderOptions($genderOption) {
+    civicrm_api3('OptionValue', 'get', [
```

Comment:
Are we sure this option would be present on all installations? For example it is not present in my local installation. In that case, it may be better to split the API calls, one to check if it exists, the other to disable.

Reply:
I thought that this is implicit on Chained API calls, like if the Get returns nothing, nothing will be disabled. No?

Reply:
I think it will throw an exception when making the create call to disable because by then the Id will not exist. I've seen that type of exception somewhere. You can also test on local by removing the option temporarily/

Reply:
Since you confirmed it works fine after testing locally, Approving this then.